packages/css-framework/src/objects/_wrapper.scss

```diff
@@ -1 +1,9 @@
-// Wrapper Object
\ No newline at end of file
+.container {
+  max-width: 1440px;
```

Comment:
Can this be a variable?

Reply:
I've actually just removed this as the max width is for the component lib contain and isn't needed in the docs site apparently.